docker_entrypoint

```diff
@@ -62,7 +62,8 @@ set -x
 
 if [[ "${IS_LITESTREAM_ENABLED}" == 'true' ]]; then
 
-  export readonly DB_PATH="/app/data/store.db"
+  readonly DB_PATH="/app/data/store.db"
+  export DB_PATH
```

Comment:
We can delete this line. If it wasn't actually exporting before, we don't need to export it.